### DISCUSSION TITLE
Journal bearing clearance and rotor stability

### DISCUSSION THREAD

**Discussion by arnal88** [petrobras/ross#927] [Q&A]
Hi,

I have a question regarding the influence of bearing clearance on rotor stability.
 
I've written the following code as an example:
```from pathlib import Path
import ross as rs
import numpy as np


#--- Material ---
steel = rs.Material(name="Steel", rho=7800, E=210e9, Poisson=0.3)


#--- Elements ---

L =   np.array([0.1, 0.1, 0.5, 0.1, 0.1])
N = len(L)
o_d = np.array([0.04, 0.04, 0.1, 0.04, 0.04])

elements = [
    rs.ShaftElement(
        L=L[i],
        idl=0,
        odl=o_d[i],
        material=steel
    )
    for i in range(N)
]


#--- Bearings ---

C = 3e-5  # Clearance
W = 30    # Load

print("Bearing clearance: " + str(round(C*1000,3)) + " mm\nBearing load: " + str(W) + " N")

bearing_1 = rs.BearingFluidFlow(
    n=1,
    nz=10,
    ntheta=30,
    length=0.1,
    omega=np.linspace(20, 250, 6),
    p_in=0,
    p_out=0,
    radius_rotor=0.02,
    radius_stator=0.02+C,
    visc=1e-3,
    rho=1000,
    load=W,
)

# Copy bearing to decrease compute time
bearing_2 = rs.BearingElement(
    n=4,
    frequency=bearing_1.frequency,
    kxx=bearing_1.kxx,
    kxy=bearing_1.kxy,
    kyx=bearing_1.kyx,
    kyy=bearing_1.kyy,
    cxx=bearing_1.cxx,
    cxy=bearing_1.cxy,
    cyx=bearing_1.cyx,
    cyy=bearing_1.cyy,
)

bearings = [bearing_1, bearing_2]


#--- Rotor ---
rotor = rs.Rotor(
    shaft_elements=elements,
    bearing_elements=bearings,
)

rotor.plot_rotor().show()


#--- Campbell Diagram ---
speed_range = np.linspace(20, 250, 24)
campbell = rotor.run_campbell(speed_range)

campbell.plot(
	title="Load = "+str(W)+" N  /  Clearance = "+str(round(C*1000,3))+" mm"
).show()


#--- Modal analysis ---
modal = rotor.run_modal(150)

print(f"1st natural frequency:\n {modal.wd[0]}")
print(f"Logarithmic decrement:\n {modal.log_dec[0]}")
```

This is a very simple shaft with 2 hydrodynamic journal bearings:
![image](https://user-images.githubusercontent.com/16865032/202483802-c1fa1bbb-0252-417f-8609-505bf5496d71.png)

When I run the code, I get the following Campbell diagram (I've zoomed on lower frequencies):
![image](https://user-images.githubusercontent.com/16865032/202488402-87f5b4aa-17e6-4497-8c1f-16d90d2218f2.png)

The first natural frequencies are located at about half the rotation speed, which seems to be typical of oil whirl in journal bearings.

I've read in the literature that decreasing load reduces stability, and that increasing clearance reduces stability as well.

Indeed, I get the following results (at 150 rad/s):
 - Load = 30N  ==>  log decrement = 0.71
 - Load = 10N  ==>  log decrement = 0.47

But increasing clearance actually improves stability:
 - Clearance = 0.03mm  ==>  log decrement = 0.71
 - Clearance = 0.05mm  ==>  log decrement = 1.29

What is going wrong with that?
Why increasing bearing clearance actually increases stability instead of decreasing it?
Is there something that I've not correctly understood on hydrodynamic journal bearing stability?
Or does it come from the code (model limitations or bug)?

Thanks!

**Comment by LeifParr** [comment]
Hi.
I think that the concept of stability from classical rotor dynamic theory is also related to the shaft speed, e.g. the shaft speed above which the damping of the first forward bending mode goes below a certain threshold.
This is described in this article:
https://dyrobes.com/wp-content/uploads/2015/09/Instability-Threshold-and-Stability-Boundaries-of-Rotor-Bearing-Systems_linked.pdf

Also, it is my experience that the effect of bearing load and clearance should be investigated for each real-world application. For example, I have seen a case where increasing bearing clearance of a hydropower rotor bearing was beneficial, as it lowered the critical speed and increased the damping, which made the resulting vibration lower when passing through the critical speed.  But in other cases, this is not beneficial, as we try to keep the critical speed above the operating range.
